plugins/experimental/remap_stats/remap_stats.cc

```diff
@@ -178,8 +181,8 @@ handle_txn_close(TSCont cont, TSEvent event ATS_UNUSED, void *edata)
         remap = get_effective_host(txn);
       }
 
-      if (remap.empty()) {
-        remap = unknown;
+      if (nullptr == remap) {
+        remap = const_cast<char *>(unknown);
```

Comment:
Rather than `const_cast`, could `remap` be declared as `char const *`?

Reply:
The original code was a hack to try to dual purpose a pointer (shorter code always runs faster rite?). Introducing another pointer allows remap to be used for aliasing only which cleans things up.

Reply:
I don't understand your response. What other pointer? I don't see what that has to do with declaring `remap` as `char const *`?

Reply:
i pushed up code that should be more clear.